src/backend-api/src/main/kotlin/org/icpclive/api/Settings.kt

```diff
@@ -124,7 +124,8 @@ data class ImageTickerSettings(
 
 @Serializable
 @SerialName("clock")
-data class ClockTickerSettings(override val part: TickerPart, override val periodMs: Long) : TickerMessageSettings() {
+data class ClockTickerSettings(override val part: TickerPart, override val periodMs: Long, val timeZone: String = ""
```

Comment:
Please use null instead of empty string for the absence of value. 

Also, about formating. It should be either each argument on separate line, or whole header on single line. Probably first would be better here now.

Reply:
Now it's null + fix formating